cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -812,4 +813,62 @@ public static List<String> generateAllServiceDnsNames(String namespace, String s
 
         return dnsNames;
     }
+
+    /**
+     * Validate cpu and memory resources.
+     * Early resources validation avoids triggering any pod operation with invalid configuration.
+     * 
+     * For both cpu and memory, this method checks that request is greater than zero, 
+     * limit is greater than zero and limit is greater than or equal to the request.
+     * 
+     * @param resources Resources configuration.
+     * @param owner Resources owner.
```

Comment:
Perhaps you can be a bit clear in the comment what the `owner` is or rename it to something like `resourcePath`? Both here in the private method. TBH, initially I was quite confused what does owner reference have to do with this.